src/Encoder.c

```diff
@@ -4373,7 +4373,8 @@ ZYDIS_EXPORT ZyanStatus ZydisEncoderEncodeInstructionAbsolute(ZydisEncoderReques
                     }
                     if ((rel_info->accepts_branch_hints) &&
```

Comment:
We cannot depend on just `accepts_branch_hints` here and in tests. It's only for `BRANCH_TAKEN` and `BRANCH_NOT_TAKEN`. This means `BND` fix covers BND+JCC but doesn't cover `BND+JMP` which doesn't accept branch hints.

Reply:
BND+CALL is affected as well.